package.json

```diff
@@ -11,11 +11,13 @@
     "start": "node src/web",
     "start:dev": "npm run build && nodemon src/web",
     "test": "npm-run-all build test:lint test:unit -p -r start test:browser",
-    "test:lint": "standard --env mocha",
+    "test:lint": "standard",
     "test:unit": "nyc tape -r './src/test/unit/setup' 'src/@(web|config)/**/*.test.js'",
-    "test:acceptance": "mkdir -p build/reports && cucumber-js src/test/acceptance/features/ -f json:build/reports/cucumber-report.json",
-    "test:acceptance:report": "node src/test/cucumber-report.js",
-    "test:smoke": "mocha 'src/test/smoke/test-suite.js'",
+    "test:acceptance": "mkdir -p build/reports && cross-env TESTS=acceptance cucumber-js src/test/acceptance/features/ --require 'src/test/common/steps/*.js' --require 'src/test/acceptance/config' -f json:build/reports/acceptance-report.json",
+    "test:acceptance:report": "node src/test/acceptance-report.js",
+    "test:smoke": "npm-run-all -s test:smoke:test test:smoke:report",
+    "test:smoke:test": "mkdir -p build/reports && cross-env TESTS=smoke cucumber-js src/test/smoke/features/ --require 'src/test/common/steps/*.js' --require 'src/test/smoke/config' -f json:build/reports/smoke-report.json",
```

Comment:
Some of these scripts are getting quite long, should we consider moving to separate files? Or maybe it doesn't matter?

Reply:
I think that's my thought, so let's leave it for now.